src/Docker/Runner/Limits.php

```diff
@@ -140,19 +140,20 @@ private function getInstanceCpuLimit()
         throw new ApplicationException("cpu_count is not set in parameters.yml");
     }
 
-    private function getNodeTypeMultiplier(?string $containerType): int
+    private function getNodeTypeMultiplier(?string $containerType): float
     {
         switch ($containerType) {
             case null:
+            case 'xsmall':
+                return 0.5;
             case 'small':
                 return 1;
             case 'medium':
                 return 2;
             case 'large':
-                return 4;
-            case 'xlarge':
                 // see https://github.com/keboola/job-queue-daemon/blob/7af7d3853cb81f585e9c4d29a5638ff2ad40107a/src/Cluster/ResourceTransformer.php#L34
-                return 14.2;
+                // https://keboola.atlassian.net/wiki/spaces/KB/pages/2234941476/Dynamic+backend+size+per+job+Python+R#Option-2-(preferred-by-odin):
+                return 7.1;
```

Comment:
Souvisi ten multiplier nejak s creditama v teto tabulce https://keboola.atlassian.net/wiki/spaces/KB/pages/2619768896/Python+R+Data+Science+Jobs+Backend+Billing ? (az na tu large hodnotu se to shoduje)

Reply:
Nene, souvisí s multiplierem v této tabulce https://keboola.atlassian.net/wiki/spaces/KB/pages/2234941476/Dynamic+backend+size+per+job+Python+R#Option-2-(preferred-by-odin):